janusgraph-dist/pom.xml

```diff
@@ -582,6 +587,64 @@
                         </executions>
                     </plugin>
 
+                    <plugin>
```

Comment:
This is part of the `janusgraph-release` profile which of course makes sense as that also builds the distribution archives. We should document that here: https://github.com/JanusGraph/janusgraph/blob/master/BUILDING.md#building-docker-image-for-janusgraph-server
and probably also directly above that in the sentence:
> This command generates the distribution archive [...]

It at least wasn't obvious to me directly that I had to enable that profile to build the docker image.